### PR TITLE
add load 3d node support

### PR DESCRIPTION
**Pull request by jtydhr88**
Hi,
I implemented a node to load local 3d models and output IMAGE for furthur process, such as controlnet, as reference image.
here is a demo video:
https://github.com/user-attachments/assets/703b12ed-9e60-42df-9de2-31fa1df48ce7
Need https://github.com/comfyanonymous/ComfyUI/pull/5564 as BE

### CODE REVIEW

File: src/extensions/core/load3d.ts [hunk 1]
```diff
@@ -0,0 +1,817 @@
+// @ts-strict-ignore
```
Comment (huchenlei) [inline src/extensions/core/load3d.ts]: Please enable ts-strict here.

**Review by huchenlei — CHANGES_REQUESTED** [review]

File: src/extensions/core/load3d.ts (hunk 1 above)
Comment (jtydhr88) [reply src/extensions/core/load3d.ts]: done

**Comment by jtydhr88** [comment]
add stl format support, also add normal and wireframe material support
image
image

**Comment by jtydhr88** [comment]
currently, the two browers tests failed but I am not sure it is caused by my changes, I checked another recent PR, like https://github.com/Comfy-Org/ComfyUI_frontend/pull/1537, it also has such issues @huchenlei

**Comment by huchenlei** [comment]
> currently, the two browers tests failed but I am not sure it is caused by my changes, I checked another recent PR, like #1537, it also has such issues @huchenlei

Sorry about that. It was introduced by accident. Please rebase your branch. The playwright test failure was fixed in https://github.com/Comfy-Org/ComfyUI_frontend/pull/1561

**Comment by jtydhr88** [comment]
> > currently, the two browers tests failed but I am not sure it is caused by my changes, I checked another recent PR, like #1537, it also has such issues @huchenlei
> 
> Sorry about that. It was introduced by accident. Please rebase your branch. The playwright test failure was fixed in #1561

Done

**Review by huchenlei — APPROVED** [review]

**Comment by jtydhr88** [comment]
/home/runner/work/ComfyUI_frontend/ComfyUI_frontend/src/stores/coreSettings.ts
Error:   5:3  error  'LinkReleaseTriggerMode' is defined but never used  unused-imports/no-unused-imports
One test failed on this line but likely it is caused by another change @huchenlei

**Comment by yovsac** [comment]
This is exactly what I was looking for for a while! How can I use this node??

**Comment by jtydhr88** [comment]
hi @yovsac , as you see, the FE code already merged, 
However, we are still waiting for one PR in core repo https://github.com/comfyanonymous/ComfyUI/pull/5564,
if you real need this feature, you might refer and just "copy" my code in https://github.com/comfyanonymous/ComfyUI/pull/5564 to your local env until the PR merged

**Comment by yovsac** [comment]
Thank you for your quick reply. Im using Comfyui portable. Can you tell me how I can copy the code?

**Comment by jtydhr88** [comment]
@yovsac I just realize coping code to your local env might not a good idea if you are using comfyui portable version, because I am not sure it is already included the FE code here.
However, if you really want to try this, you can follow:
1. create a py file under your ComfyUI\comfy_extras folder, named it nodes_load_3d.py
image

2. copy the code from https://github.com/comfyanonymous/ComfyUI/pull/5564/files into nodes_load_3d.py
3. edit your ComfyUI\nodes.py
image
4. go to init_builtin_extra_nodes() method, add one line inside extras_files, 
image
5. then restart your ComfyUI

Still, this method may not work as your local env might not included the latest FE code.

**Comment by yovsac** [comment]
Thank you very much. I’ll try that!!

**Comment by jtydhr88** [comment]
@yovsac maybe the best idea is, could you help to go the core PR https://github.com/comfyanonymous/ComfyUI/pull/5564, leave your idea and may push it merge quickly?

**Comment by yovsac** [comment]
I left a message. Hope they hurry up!

**Pull request merged**